arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java

```diff
@@ -312,11 +313,12 @@ private void triggerNonTrivialOrNoop(BytecodeCompiler program, boolean isHalting
     OpCode opCode;
 
     if (!isHalting) {
-      mxpType = MxpType.values()[util.nextRandomInt(1, 5)]; // Type 1 to 4 excluding opCodeLast
-      opCode = getRandomOpCodeByType(mxpType);
+      mxpType =
+          MxpType.values()[util.nextRandomInt(1, 5)]; // Type 1 to 4 excluding opCodesType4Halting
+      opCode = getRandomNonHaltingOpCodeByType(mxpType);
```

Comment:
The function body of `getRandomNonHaltingOpCodeByType` didn't change. Was it about renaming it ?